src/app/zap-templates/zcl/data-model/chip/microwave-oven-mode-cluster.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <enum name="ModeTag" type="enum16">
+    <cluster code="0x005E"/>
+    <item value="0x4000" name="Normal"/>
+    <item value="0x4001" name="Defrost"/>
+  </enum>
+
+  <cluster>
+    <domain>General</domain>
+    <name>Microwave Oven Mode</name>
+    <code>0x005E</code>
+    <define>MICROWAVE_OVEN_MODE_CLUSTER</define>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <description>Attributes and commands for selecting a mode from a list of supported options.</description>
+    <globalAttribute side="either" code="0xFFFD" value="2"/>
```

Comment:
This does not match the spec.

Stopping here.  Please go through and make this PR match the spec.